src/pages/Login/My/My.scss

```
@@ -0,0 +1,253 @@
+.my-page-wrapper {
+  font-family: 'noto sans';
+  font-size: 12px;
+  margin: 0px 100px 20px 100px;
+  padding: 0px 170px;
+  .title-container {
+    .my-main-title {
+      border-bottom: 1.5px solid #333;
+      margin-bottom: 10px;
+      .title {
+        height: auto;
+        padding: 40px 0px 20px 0px;
+        font-size: 24px;
+        font-weight: 530;
+      }
+    }
+  }
+  .item-list-wrapper {
+    .item-list {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      border-bottom: 1px solid rgb(223, 223, 223);
+      .item {
+        font-size: 16px;
+        padding: 14px 40px 14px;
+        margin: 0px 14px;
+        color: #333;
+        .link-item {
+          cursor: pointer;
+        }
+      }
+    }
+  }
+  .my-info {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 30px 0px;
+    .box1 {
+      padding: 40px 0px 75px 20px;
+      margin-right: 30px;
+      .button-container {
+        padding-left: 170px;
+        margin-top: -65px;
+        .benefit-button {
+          width: 100px;
+          height: 28px;
+          line-height: 15px;
+          font-size: 12px;
+          color: #666;
+          border: 1px solid #ddd;
+          border-radius: 4px;
+          background: transparent;
+          cursor: pointer;
+        }
+      }
+      .inner1 {
+        padding: 30px 0;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        .my-logo {
+          width: 130px;
+          margin-right: 40px;
+        }
+        .text {
+          .text1 {
+            font-size: 17px;
+          }
+          .text2 {
+            font-size: 17px;
+          }
+        }
+      }
+    }
+    .box2 {
+      width: 45%;
+      padding: 50px 0px 50px 100px;
+      margin-left: 30px;
+      .inner2 {
+        .point-list {
+          display: flex;
+          text-align: center;
+          flex-wrap: wrap;
+          font-size: 10px;
+          .point-item {
+            width: 33.3%;
+            &:nth-child(4) {
+              margin-top: 50px;
+            }
+            &:nth-child(5) {
+              margin-top: 50px;
+            }
+            &:nth-child(6) {
+              margin-top: 50px;
+            }
```

- 하나로 묶어서 할당할 수 있겠네요

---

```suggestion
            .point-item-btm {
            margin-top: 50px;
          }
```
아래 3개의 아이템은 .point-item-btm 클래스로 묶어서 한번에 스타일링 했습니다.